control-plane-operator/controllers/hostedcontrolplane/cloud/kubevirt/manifests.go

```diff
@@ -0,0 +1,71 @@
+package kubevirt
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func CCMServiceAccount(ns string) *corev1.ServiceAccount {
+	return &corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kubevirt-cloud-controller-manager",
+			Namespace: ns,
+		},
+	}
+}
+
+func CCMRole(ns string) *rbacv1.Role {
+	return &rbacv1.Role{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kubevirt-cloud-controller-manager",
+			Namespace: ns,
+		},
+	}
+}
+
+func CCMRoleBinding(ns string) *rbacv1.RoleBinding {
+	return &rbacv1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kubevirt-cloud-controller-manager",
+			Namespace: ns,
+		},
+	}
+}
+
+func CCMDeployment(ns string) *appsv1.Deployment {
+	return &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "cloud-controller-manager",
+			Namespace: ns,
+		},
+	}
+}
+
+func CCMConfigMap(ns string) *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "cloud-config",
+			Namespace: ns,
+		},
+	}
+}
```

Comment:
should we had the `kubevirt-` prefix to these object names?

Reply:
Yes, changed
Thanks